### PR TITLE
fifo write and read

### PR DESCRIPTION
**Pull request by behzadjoudat**
fifo write and read from cpu and to the cpu works in simulation of the loopback but not sure if the data sent is sam as the data received!

### CODE REVIEW

File: hardware/include/iob_pcie_swreg_def.vh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+//This file was generated by script mkregs.py
```
Comment (jjts) [inline hardware/include/iob_pcie_swreg_def.vh:1]: this is a generated file , cannot be committed

File: hardware/include/iob_pcie_swreg_gen.vh [hunk 2]
```diff
@@ -0,0 +1,79 @@
+//This file was generated by script mkregs.py
```
Comment (jjts) [inline hardware/include/iob_pcie_swreg_gen.vh:1]: generated  file

File: hardware/include/inst_tb.vh [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
+assign PCIE_CHNL_RX_OFF_IF = PCIE_CHNL_TX_OFF_IF;
+assign PCIE_CHNL_RX_DATA_IF = PCIE_CHNL_TX_DATA_IF;
+assign PCIE_CHNL_RX_DATA_VALID_IF = PCIE_CHNL_TX_DATA_VALID_IF;
 assign PCIE_CHNL_TX_ACK_IF = PCIE_CHNL_RX_ACK_IF;
 assign PCIE_CHNL_TX_DATA_REN_IF = PCIE_CHNL_RX_DATA_REN_IF;
 
 
-`IOB_CLOCK(PLD_CLK_i, 16)
-  
+
```
Comment (jjts) [inline hardware/include/inst_tb.vh:12]: since you deleted the PLD clock, how can you simulate?

**Comment by jjts** [comment]
was this code simulated ?

File: hardware/include/iob_pcie_swreg_def.vh (hunk 1 above)
Comment (behzadjoudat) [reply hardware/include/iob_pcie_swreg_def.vh:1]: THE last changes we have made together you said in this case we have to have these

File: hardware/include/inst_tb.vh (hunk 3 above)
Comment (behzadjoudat) [reply hardware/include/inst_tb.vh:13]: it is simulated !

**Pull request merged**